src/test/java/homework/FruiteShopUnitTest.java

```diff
@@ -98,35 +100,95 @@ public void setUp() throws Exception {
         }
-@After
-public void tearDown() throws Exception {
+
+        @After
+        public void testDown() throws Exception {
         System.out.println("after after class");
         fruitShop = null;
         }
-@Test
-public void testAllFresh() {
+
+        @Test
+        public void testAllFresh() {
         List<Fruit> expected = new ArrayList<>();
         expected.add(apple);
         expected.add(pear);
         List<Fruit> actual = fruitShop.allFresh(LocalDate.of(2019,1,20));
         assertEquals(expected,actual);
         System.out.println("All Fresh");
         }
-@Test
-public void testAllFreshNull(){
+
+        @Test
+        public void testAllFreshNull(){
         List<Fruit> actual = fruitShop.allFresh(LocalDate.of(2020,1,4));
         assertTrue(actual.size()==0);
         }
-@Test
-public void testAllFruitOfFruitType(){
+
+        @Test
+        public void testAllFruitOfFruitType(){
+        List<Fruit> expected = new ArrayList<>();
+        expected.add(apple);
+//        expected.add(pear);
+        assertEquals(expected, fruitShop.allFruitOfFruitType(FruitType.APPLE));
         System.out.println("AllFruitType");
         }
-@Test
-public void testCheckByExpireDate() {
+
+        @Test
+        public void testAllFruitOfFruitType2() {
+                List<Fruit> expected = new ArrayList<>();
+                expected.add(strawberry);
+                assertEquals(expected, fruitShop.allFruitOfFruitType(FruitType.STRAWBERRY));
+        }
+
+        @Test
+        public void testCheckByExpireDate() {
+        List<Fruit> expected = new ArrayList<>();
+//      expected.add(apple);
+        expected.add(pear);
+        List<Fruit> actual = fruitShop.checkByExpireDate(30);
+        assertEquals(expected, actual);
         }
-@Test
-public void testAllFreshAndFruitType() {
+
+        @Test
+        public void testCheckByExpireDate2() {
+                List<Fruit> expected = new ArrayList<>();
+                expected.add(apple);
+                expected.add(pear);
+                expected.add(orange);
+                expected.add(strawberry);
+//              expected.add(apple);
+                List<Fruit> actual = fruitShop.checkByExpireDate(29);
+                assertEquals(expected, actual);
+        }
+
+        @Test
+        public void testAllFreshAndFruitType() {
+        List<Fruit> expected = new ArrayList<>();
+        expected.add(apple);
+        List<Fruit> actual = fruitShop.allFreshAndFruitType(FruitType.APPLE, LocalDate.of(2019, 1, 21));
+        assertEquals(expected, actual);
+        }
+
+        @Test
+        public void testAllFreshAndFruitType2() {
+                List<Fruit> expected = new ArrayList<>();
+                expected.add(pear);
+//                expected.add(apple);
+                List<Fruit> actual = fruitShop.allFreshAndFruitType(FruitType.PEAR, LocalDate.of(2019, 1, 21));
+                assertEquals(expected, actual);
```

Comment:
You have two identical tests with just the other parameters, I think it should not be done